src/Components/Categories/Categories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Categories.css';
+
+function Categories() {
+  return (
+    <div>
+      <div className="container catContainer">
+        <div className="row">
+          <h2 className="col-12 catTitle my-5 px-5">Check status</h2>
+        </div>
```

Comment:
- [ ] Kindly ensure that you make this a button and not a header 2.

![image](https://user-images.githubusercontent.com/86318284/193224761-54ea071d-8a4e-4f4a-bae6-6db52a2268f9.png)